node_js/metrics/crashes_checks/services/ElasticSearchService.js

```diff
@@ -29,5 +33,43 @@ module.exports = {
     };
     const url = `/*-error-*/_count`;
     return api.post(url, body);
-  }
+  },
+  getTransactionMetrics: (projectName, environment) =>
+    api.post('/apm-*-transaction-*/_search?size=0', {
+      aggs: {
+        requests: {
+          filter: {
+            bool: {
+              filter: [
+                ...projectFilters(projectName, environment),
+                {
+                  range: {
```

Comment:
This range is repeated in both requests to the service, as said in other comment, you could extract just the range into a smaller piece and call it in every method just as `range: weekAgo()` or `range: monthAgo()`